protocol/synthetix/contracts/modules/core/PoolModule.sol

```diff
@@ -228,6 +228,66 @@ contract PoolModule is IPoolModule {
         emit PoolNameUpdated(poolId, name, msg.sender);
     }
 
+    /**
+     * @inheritdoc IPoolModule
+     */
+    function enablePoolCollateralDelegation(uint128 poolId, address collateral) external override {
+        Pool.Data storage pool = Pool.loadExisting(poolId);
+        Pool.onlyPoolOwner(poolId, msg.sender);
+
+        pool.disabledCollaterals[collateral] = false;
+
+        emit PoolCollateralEnabled(poolId, collateral, msg.sender);
+    }
+
+    /**
+     * @inheritdoc IPoolModule
+     */
+    function disablePoolCollateralDelegation(uint128 poolId, address collateral) external override {
+        Pool.Data storage pool = Pool.loadExisting(poolId);
+        Pool.onlyPoolOwner(poolId, msg.sender);
+
+        pool.disabledCollaterals[collateral] = true;
+
+        emit PoolCollateralDisabled(poolId, collateral, msg.sender);
+    }
+
+    /**
+     * @inheritdoc IPoolModule
+     */
+    function isDelegationEnabledByPool(
+        uint128 poolId,
+        address collateral
+    ) external view override returns (bool) {
+        return !Pool.load(poolId).disabledCollaterals[collateral];
+    }
+
+    /**
+     * @inheritdoc IPoolModule
+     */
+    function setPoolCollateralIssuanceRatioD18(
```

Comment:
Unsure if we need D18 in the function name here? (Don't think we're doing this elsewhere.) Good to have it clarify in the parameter name though.